accounts/migrations/0002_delete_userprofile.py

```diff
@@ -0,0 +1,16 @@
+# Generated by Django 4.0.3 on 2022-05-27 22:46
```

Comment:
Are you sure we need this migrations file? I don't see a new model in this pr

Reply:
It is main problem the file created auttomatically